build_release.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import os
+from runner.models.anthropic_codegen import build_release_guidelines as build_anthropic_guidelines
+from runner.models.openai_codegen import build_release_guidelines as build_openai_guidelines
+
+def main():    
+    os.makedirs("dist", exist_ok=True)
+    
+    # Generate rules (we could do more model-specific ones in the future)
+    # Using a very specific filename here to make it clear for AI usage what this is.
+
+    with open("dist/anthropic_convex_rules.txt", "w") as f:
+        f.write(build_anthropic_guidelines())
+    
+    with open("dist/openai_convex_rules.txt", "w") as f:
+        f.write(build_openai_guidelines())
```

Comment:
Alternative to making it a function:
```suggestion
from runner.models.anthropic_codegen import ANTHROPIC_CONVEX_GUIDELINES
from runner.models.openai_codegen import OPENAI_CONVEX_GUIDELINES

def main():    
    os.makedirs("dist", exist_ok=True)
    
    # Generate rules (we could do more model-specific ones in the future)
    # Using a very specific filename here to make it clear for AI usage what this is.

    with open("dist/anthropic_convex_rules.txt", "w") as f:
        f.write(ANTHROPIC_CONVEX_GUIDELINES)
    
    with open("dist/openai_convex_rules.txt", "w") as f:
        f.write(OPENAI_CONVEX_GUIDELINES)
```

Reply:
Ahh.. you see this is where my lack of Python knowledge comes in. I didnt see that `if __name__ == "__main__":` section in `guidelines.py` :( 

So I have duplicated a bunch of work with `build_release.py`

Im going to leave that code in `guidelines.py` for now and keep my build_release.py.

Ill fix up that code I broke there tho, apologies.

Reply:
Okay this should now be okay @ianmacartney ?